addons/medical_gui/gui.hpp

```diff
@@ -224,6 +224,7 @@ class ACE_Medical_Menu {
             w = QUOTE(POS_W(1.5));
             h = QUOTE(POS_H(1.5));
             color[] = {1, 1, 1, 1};
+            colorActive[] = {1,1,1,0.8};
```

Comment:
```suggestion
            colorActive[] = {1, 1, 1, 0.8};
```

Reply:
Had already pushed an update to change this, guess it didn't go through